src/methods.cpp

```diff
@@ -191,6 +194,18 @@ extern "C"
         return PyUnicode_FromString(scorepy::abspath(path).c_str());
     }
 
+    static PyObject* manual_finalize(PyObject* self, PyObject* args)
```

Comment:
Would be also a valid name. I'll rename it.

I wasn't sure if you want me to put it into a user interface, since `get_experiment_dir_name` also hasn't one.